project_task_pull_request_state/models/project_project.py

```diff
@@ -0,0 +1,14 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import fields, models
+
+
+class ProjectState(models.Model):
```

Comment:
Model name follows the filename commonly, and since this model is a well-known one there's no point in having different name.

Reply:
fixed